lib/oli/delivery/attempts/activity_lifecycle/evaluate.ex

```diff
@@ -460,6 +471,39 @@ defmodule Oli.Delivery.Attempts.ActivityLifecycle.Evaluate do
     end
   end
 
+  defp persist_client_evaluations(part_inputs, client_evaluations, roll_up_fn, replace) do
+    case client_evaluations
+         |> Enum.map(fn %{
+                          attempt_guid: attempt_guid,
+                          client_evaluation: %ClientEvaluation{
+                            score: score,
+                            out_of: out_of,
+                            feedback: feedback
+                          }
+                        } ->
+      {:ok,
+        %Oli.Delivery.Evaluation.Actions.FeedbackActionResult{
+          type: "FeedbackActionResult",
+          attempt_guid: attempt_guid,
+          feedback: feedback,
+          score: score,
+          out_of: out_of
+        }}
+    end)
+         |> (fn evaluations -> {:ok, evaluations} end).()
+         |> persist_evaluations(part_inputs, roll_up_fn, replace) do
+      {:ok, results} ->
+        results
+
+      {:error, error} ->
+        Logger.debug("error inside apply_client_evaluation: #{error}")
```

Comment:
might want to use this recent addition: https://github.com/Simon-Initiative/oli-torus/blob/master/lib/oli/utils.ex#L22

Reply:
Done